samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -20,6 +20,7 @@ namespace Azure.AI.DocumentTranslation
     /// <summary> The DocumentTranslation service client. </summary>
     public partial class DocumentTranslationClient
     {
+        /// <Pipeline> The HTTP pipeline for sending and receiving REST requests and responses. </Pipeline>
```

Comment:
either works, but I would expect to see
```c#
 /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
```

Reply:
Strange, I wonder why it did that, let me look.

Reply:
Ah, I called it wrong.